Makefile

```diff
@@ -50,6 +51,7 @@ GO_FILES := $(shell find . -name '*.go' | grep -v -E '$(GO_EXCLUDE)')
 # Environment for tests, the directory containing istio and deps binaries.
 # Typically same as GOPATH/bin, so tests work seemlessly with IDEs.
 export ISTIO_BIN=${GO_TOP}/bin
+DEP:=${ISTIO_BIN}/dep
```

Comment:
DEP ?= ... ( so it can be replaced )

Reply:
I'm not so sure that's desirable.  Generally speaking if someone uses their own or a different tool then they'll be more apt to break the build for other people or systems who are using the canonical tool.  This philosophy generally works better when the tools are locked down to a particular version, which granted we're not really getting with "dep" since currently each person can still be getting a different version of the tool depending on when "go get" was run.  The flip side of the argument is that a tool might have a different name on another OS (e.g., Mac of Windows), though dep isn't a good example of that either since the build is downloading the tool.